<launch>    

    <param name="/use_sim_time" value="true" />

    <node pkg="ndt_feature" type="ndt_feature2d_view" name="ndt_feature2d_view" output="screen">
      <param name="laser_topic" value="vmc_navserver/laserscan"/>
      <param name="resolution" value="1"/>
    </node>

    <node pkg="rviz" type="rviz" name="rviz" args="$(find ndt_feature)/launch/ndt_feature_view_rviz.vcg" />

     <node pkg="tf" type="static_transform_publisher" name="world_map_frame" args="0 0 0 0 0 0 /world /map 10"/>
     <node pkg="tf" type="static_transform_publisher" name="world_laser_frame" args="0 0 0 0 0 0 /world /base_laser_link 10"/>

</launch>
